### ISSUE TITLE
Spotify iOS demo project "SPTLoginSampleApp" not redirecting back on the app after auth. - Objective-C.

### ISSUE THREAD

**Issue by rajatk001** [spotify/ios-sdk#313]
Hello all,

I am facing an issue regarding, when the demo app redirects to the spotify app, a tick mark appears and the app remains on the spotify app.

**Expected Behaviour:** After authentication on the spotify app, the demo app should be open.

**Comment by Abhishek12392** [comment]
I am facing this issue too.

**Comment by Seetu211** [comment]
yes, i am facing the same issue "I am facing an issue regarding, when the demo app redirects to the spotify app, a tick mark appears and the app remains on the spotify app."
